app/view_objects/suggested_search_link.rb

```diff
@@ -34,4 +35,11 @@ def suggested_search_link_parameters(radius:)
       .reject { |k, _v| %w(lat lng rad loc lq).include?(k) }
       .merge("l" => 2)
   end
+
+  def text_course_text
+    return "non-salaried course" if including_non_salaried
```

Comment:
Not too sure about the wording...

non-salaried

Reply:
If Paul Genders is happy, then I'm happy

Reply:
pretty sure that terminology was leaked 

https://github.com/DFE-Digital/find-teacher-training/pull/226#discussion_r398030780

Reply:
![image](https://user-images.githubusercontent.com/470137/78788237-6b986100-79a3-11ea-84ed-22f7206ae5f8.png)

Until this is bottom out it, it is not going anywhere far...


`non-salaried courses` does not mean `Courses with and without salary`
The suggested search number does not tally up/follow through 

LHS url
https://s121d02-3029-find-as.azurewebsites.net/results?fulltime=false&funding=8&hasvacancies=true&l=1&lat=55.953252&lng=-3.188267&loc=Edinburgh%2C+UK&lq=Edinburgh%2C+UK&parttime=false&qualifications[]=QtsOnly&qualifications[]=PgdePgceWithQts&qualifications[]=Other&rad=5&senCourses=false&sortby=2

Reply:
I think this is probably a bug in one of the queries rather than an issue with copy or semantics there of. I'll do another card to look at this so as we can just get the copy sorted in this PR.